server/index.js

```diff
@@ -1,22 +1,41 @@
 const express = require("express");
 const cors = require("cors");
-
+const products = require("./src/data/places.json");
 const app = express();
 const PORT = 8800;
 
 app.use(cors());
+app.use(express.json());
+
+app.get("/places/:id", (req, res) => {
+  const placeId = parseInt(req.params.id);
+  const place = products.find((p) => p.id === placeId);
+
+  if (place) {
+    res.json(place);
+  } else {
+    res.status(404).json({ error: `Place with ID ${placeId} not found.` });
+  }
+});
+
```

Comment:
If you like I feel this piece of code can be placed in the routes/places.js file.  @Konzisam Correct me if I am wrong.